chat.js

```diff
@@ -777,6 +777,10 @@ class CommandContext {
 						this.errorReply(`Because moderated chat is set, your account must be at least one week old and you must have won at least one ladder game to speak in this room.`);
 						return false;
 					}
+					if (room.modchat === 'trusted') {
+						this.errorReply(`Because moderated chat is set, your account must be staff in a public room or have global auth to speak in this room.`);
```

Comment:
"Global auth" isn't something I want to appear in an error message.

How about "have a global rank"?